myia/pipeline/standard.py

```diff
@@ -86,6 +86,8 @@
     np.tanh: operations.array_tanh,
     np.sum: operations.sum,
     np.prod: operations.prod,
+    public_api.myia_random_initialize: operations.random_initialize,
+    public_api.myia_random_uniform: operations.random_uniform,
```

Comment:
I'm not sure what `public_api` is for. `myia.operations` already is a public API that the user is allowed to import from.

Reply:
Oh, ok, I did not know. So, user could just directly import operations `random_initialize` and `random_uint` and use them as functions in python code.

Reply:
Yes, exactly.